src/components/tasks/card/card.module.scss

```diff
@@ -1,60 +1,101 @@
-.card {
-  display: flex;
-  flex-direction: column;
-  padding: 1rem;
-  margin-bottom: 2rem;
-  margin-bottom: 25px;
-  justify-content: space-between;
-  border-bottom-style: solid;
-  border-bottom-color: #d9d9d9;
-  border-bottom-width: 1px;
-}
-
-.cardItems {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-}
-
-.cardItems {
-  span {
-    display: flex;
-    flex-wrap: wrap;
-    margin: 2px;
-    align-items: center;
-    margin-top: 10px;
-  }
-}
-
-.cardTitle {
-  font-size: 1.6rem;
-  font-weight: 500;
-  color: #041187;
-}
-
-.cardSpecialFont {
-  font-size: 1.1rem;
-  font-weight: 500;
-  color: #aeaeae;
-}
-
-.cardStatusFont {
-  font-size: 1.3rem;
-  font-weight: 500;
-}
-
-.contributorImage {
-  img {
-    border-radius: 40%;
-  }
-}
-
-.cardStrongFont {
-  font-size: 1.3rem;
-  font-weight: 500;
-}
-
-.overdueTask {
-  border: solid red 2px;
-  background-color: #ffebee;
-}
+$green: #008000;
+$orange: #ffa500;
+$red: #ff0000;
+$yellow: #ecef08;
+$white: #e5e5e5;
+$black: #252525;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;

```

Comment:
Aren't we using 'px' everywhere?

Reply:
some places we are, and some places we are not using it.